src/ArcherContainer.js

```diff
@@ -85,6 +84,19 @@ function computeCoordinatesFromAnchorPosition(
   }
 }
 
+const possibleShapes: Array<ValidShapeTypes> = ['arrow', 'circle'];
+
+const getEndShapeFromStyle = (shapeObj: LineType) => {
+  if (!shapeObj.endShape) {
+    return possibleShapes[0];
+  }
```

Comment:
If no `endShape` is provided, this will return `arrow`, as `react-archer` currently does.